src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -563,7 +563,129 @@ def compute_synchrony_metrics(waveform_extractor, synchrony_sizes=(2, 4, 8), **k
     return synchrony_metrics
 
 
-_default_params["synchrony_metrics"] = dict(synchrony_sizes=(0, 2, 4))
+_default_params["synchrony"] = dict(synchrony_sizes=(0, 2, 4))
+
+
+def compute_firing_ranges(waveform_extractor, bin_size_s=5, quantiles=(0.05, 0.95), unit_ids=None):
+    """Calculate firing range, the range between the 5th and 95th quantiles of the firing rates distribution
+    computed in non-overlapping time bins.
+
+    Parameters
+    ----------
+    waveform_extractor : WaveformExtractor
+        The waveform extractor object.
+    bin_size_s : float, default: 5
+        The size of the bin in seconds.
+    quantiles : tuple, default: (0.05, 0.95)
```

Comment:
I see why you chose quantile in the doc string, but technically a quantile needs to be an equal division of the data. So you could dive the data into fifths or tenths with percentiles being hundredths. It might be too annoying to change the term and I think most users would just understand, but I assume I could enter `quantiles = (0.02, 0.97)` and it would still be okay, although in this case you're allowing me to change the percentiles. No need to change, but just to keep in mind that the docs mention percentile and then the function uses quantile.

Reply:
let's use percentiles then ;)

Reply:
done